MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -513,8 +513,8 @@ private async Task<MediaInfo> GetMediaInfoInternal(
             CancellationToken cancellationToken)
         {
             var args = extractChapters
-                ? "{0} -i {1} -threads {2} -v warning -print_format json -show_streams -show_chapters -show_format"
-                : "{0} -i {1} -threads {2} -v warning -print_format json -show_streams -show_format";
+                ? "{0} -i {1} -threads {2} -v warning -print_format json -show_streams -show_chapters -show_format -protocol_whitelist \"file,https,tls,crypto,http,tcp\""
```

Comment:
What are the security implications of allow file, and some of the others? Perhaps this should be opt-in behind a checkbox somewhere.